src/varbc_gsi/GsiAircraftBiasConverter.cc

```diff
@@ -0,0 +1,130 @@
+/*
+ *  * (C) Copyright 2023 NOAA/NWS/NCEP/EMC
+ *   *
+ *    * This software is licensed under the terms of the Apache Licence Version 2.0
+ *     * which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+ *      */
```

Comment:
Seems to be some unintended ascii art (probably from auto-commenting by editor?)